src/System.Net.Security/src/System/Net/Security/SslAuthenticationOptions.cs

```diff
@@ -52,6 +52,21 @@ internal SslAuthenticationOptions(SslServerAuthenticationOptions sslServerAuthen
             CipherSuitesPolicy = sslServerAuthenticationOptions.CipherSuitesPolicy;
         }
 
+        static private SslProtocols FilterOutIncompatibleSslProtocols(SslProtocols protocols)
```

Comment:
```suggestion
        private static SslProtocols FilterOutIncompatibleSslProtocols(SslProtocols protocols)
```

(canonical ordering)